gpytorch/kernels/rbf_kernel.py

```diff
@@ -102,6 +92,9 @@ def forward_diag(self, x1, x2):
         return res
 
     def forward(self, x1, x2):
-        lengthscales = self.log_lengthscale.exp().mul(math.sqrt(2)).clamp(self.eps, 1e5)
-        diff = (x1.unsqueeze(2) - x2.unsqueeze(1)).div_(lengthscales.unsqueeze(1))
-        return diff.pow_(2).sum(-1).mul_(-1).exp_()
+        x1_, x2_ = self._create_input_grid(x1, x2)
```

Comment:
Didn't we decide we were going to pass **args in to `forward` and pass `diag=True` or `False` on to create_input_grid?

Reply:
This PR doesn't address the `diag=True` and `**args` stuff. That'll be the next PR (trying to keep the PRs small). This one just adds the `_create_grid` method.